docs/incubating/identitycenter_custom_policy_attachment.md

```diff
@@ -0,0 +1,50 @@
+---
+subcategory: "IAM Identity Center"
+---
+
+# huaweicloud_identitycenter_custom_policy_attachment
+
+Manages an Identity Center custom policy attachment resource within HuaweiCloud.
```

Comment:
add a note that only one resource can be created

Reply:
done